### PR TITLE
Add Rocky Linux profile to Anaconda

### PR DESCRIPTION
**Pull request by nazunalika**
Rocky Linux is a enterprise Linux distribution.

This PR adds Rocky to the main branch of anaconda.

Please let me know if the files or commits require extra work or information. Thank you!

### CODE REVIEW

File: data/profile.d/rocky.conf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Anaconda configuration file for Rocky Linux.
```
Comment (poncovka) [inline data/profile.d/rocky.conf:1]: Please, update also the `data/profile.d/Makefile.am` file. That should fix the failing tests.

**Review by poncovka — CHANGES_REQUESTED** [review]
Otherwise, it looks good to me.

**Comment by jkonecny12** [comment]
Hi, welcome to the family :D :D.

File: data/profile.d/rocky.conf (hunk 1 above)
Comment (nazunalika) [reply data/profile.d/rocky.conf:1]: Hello - I added a rocky.conf line to `Makefile.am` as suggested. Let me know if you need anything else!

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**